schemas/yaml/Language.yml

```diff
@@ -150,6 +150,18 @@ properties:
           minimumProperties: 1
           properties:
 
+            AES:
+              title: 'Agglomerated Endangerment Scale'
+              type:  string
+              enum:
+              - Not Endangered
+              - Threatened
+              - Shifting
+              - Moribund
+              - Nearly Extinct
+              - Extinct
+              description: 'The Agglomerated Endangerment Scale status of this Demographic data.'
```

Comment:
Can you add a link in this description to the page where the user can read more about AES?